### PR TITLE
feat: ignoring genesis transactions before sending them to the wallet…

### PR DESCRIPTION
**Pull request by andreabadesso**
We are already doing this on the wallet service, but the wallet service will throw an error, causing the daemon to stop sending blocks.

With this PR, the daemon will not even send the transaction if it is a genesis tx

### CODE REVIEW

**Review by msbrogli — APPROVED** [review]

File: src/utils.ts [hunk 1]
```diff
@@ -39,6 +39,20 @@ import logger from './logger';
 
 dotenv.config();
 
+export const IGNORE_TXS = {
```
Comment (msbrogli) [inline src/utils.ts:42]: I don't like having these hashes here. Maybe move to the constants file?

Comment (pedroferreira1) [reply src/utils.ts:42]: Do we have the wallet-lib installed here? I think we should have the genesis hashes there and use it in the wallets. We already have this in the wallet-service, explorer and daemon

File: src/utils.ts [hunk 2]
```diff
@@ -315,8 +338,12 @@ export async function* syncToLatestBlock(): AsyncGenerator<StatusEvent> {
       for (const key of Object.keys(uniqueTxs)) {
       const preparedTx: PreparedTx = prepareTx(uniqueTxs[key]);
 
+
       try {
-        const sendTxResponse: ApiResponse = await sendTx(preparedTx);
+        const sendTxResponse: ApiResponse = await sendTx({
+          ...preparedTx,
+          height: preparedBlock.height,
```
Comment (pedroferreira1) [inline src/utils.ts:345]: Why did you add this key?

**Review by pedroferreira1 — APPROVED** [review]

File: src/utils.ts (hunk 1 above)
Comment (andreabadesso) [reply src/utils.ts:42]: We don't, but I agree that would be a good idea.

I'll merge and open an issue to refactor it when it's ready

File: src/utils.ts (hunk 2 above)
Comment (andreabadesso) [reply src/utils.ts]: This was added by mistake on this branch, it is part of some changes I am doing to the sync daemon to handle reorgs

Removed it from last commit and force pushed it.
